webapp/python/README.md

```diff
@@ -15,29 +20,54 @@
 2. Start your browser and make it point to the url `http://<host>:8000`
-   
-   To make it reachable by the Collabora Online server use as `<host>` the IP address of the machine where the Django 
-   server is running. In case the Django server can't be reached you could also need to open the port 8000 on the firewall. 
-   
+
+Or with HTTPS after configuring the certificates (see below)
+
+1. Start the Django development server: `run_https.sh`
+2. Start your browser and make it point to the url `https://<host>:8000`
+
+   To make it reachable by the Collabora Online server use as `<host>` the IP address of the machine where the Django
+   server is running. In case the Django server can't be reached you could also need to open the port 8000 on the firewall.
+
    When both the Collabora Online server and the Django server belong to the same LAN `<host>` should be a local IP address.
-   
-   In case the Collabora Online server is on a remote host on the internet, `<host>` must be the public IP address of 
+
+   In case the Collabora Online server is on a remote host on the internet, `<host>` must be the public IP address of
    the machine where the Django example is running.
-3. In the form text field enter the address for your Collabora Online server including the protocol scheme, 
+3. In the form text field enter the address for your Collabora Online server including the protocol scheme,
    then click on the `Load Collabora Online` button.
-4. An instance of Collabora Online should appear into an iframe embedded in the same page, 
+4. An instance of Collabora Online should appear into an iframe embedded in the same page,
    showing a text document loaded with content a simple `Hello world!` text line.
-5. Try to type in some more text ( e.g. `Hi!`) and then perform a `Save` action. 
+5. Try to type in some more text ( e.g. `Hi!`) and then perform a `Save` action.
    In the console where the Django server is running you should get the following output:
    * `PutFile: file id: 1`  - the PutFile wopi endpoint has been triggered
    * `Hello World! Hi!` - the updated file content has been successfully received
-    
 
+## Certificates
+
+It is highly recommended to setup TLS certificates for https.
+
+If you don't have a key pair, I recommend using
+[minica](https://github.com/jsha/minica) to generate a self-signed
+one.
+
+**THIS IS ONLY FOR TEST AND DEVELOPMENT. NEVER USE SELF SIGNED
+CERTIFICATE IN A PRODUCTION ENVIRONMENT**
+
+Then set the environment to indicate where to load the certificate from.
+
+- `SSL_KEY_FILE` contains the path to the private key. If you used
+  the `minica` tool mentionned above, it's the path to the
+  `minica-key.pem` file.
+- `SSL_CRT_FILE` contains the path to the public certificate. If you used
+  the `minica` tool mentionned above, it's the path to the
+  `minica.pem` file.
+- `DISABLE_TLS_CERT_VALIDATION` set to `1` if you use certificates that do
```

Comment:
Could be in `webapp/python/run_https.sh` or derived from `settings.DEBUG`?

Reply:
actually I don't want to code this anywhere. It has to be done by the user deliberately.

I use `direnv` for that purpose. It's the same with the Node examples.